de.dlr.sc.virsat.model.extension.tests.test/src/de/dlr/sc/virsat/build/validator/core/DvlmLatestConceptValidatorTest.java

```diff
@@ -61,15 +59,21 @@ public void testValidateLatestConcept() throws CoreException, IOException {
 		IConfigurationElement[] concepts = registry.getConfigurationElementsFor(CONCEPT_EXTENSION_POINT_ID);
 		
 		ActiveConceptConfigurationElement acElement = ActiveConceptConfigurationElement.getPropperAddActiveConceptConfigurationElement(concepts, TEST_CONCEPT_ID);
-		Command command = acElement.createAddActiveConceptCommand(domain, repo);
+		Command command = acElement.createAddActiveConceptCommand(domain, repository);
 		domain.getCommandStack().execute(command);
 
-		assertTrue("concept is up to date", validator.validate(repo));
+		assertTrue("concept is up to date", validator.validate(repository));
 		// we decrease the version of the concept
-		for (Concept concept : repo.getActiveConcepts()) {
-			concept.setVersion("0.001");
+		for (Concept concept : repository.getActiveConcepts()) {
+			editingDomain.getCommandStack().execute(new RecordingCommand(editingDomain) {

```

Comment:
We should consider a change that it is easier to apply a change to the model, without writing all the boilerplate code. A lambda expression on the generic test class could be considered.